libbpf-tools/memleak.h

```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+
+#ifndef __MEMLEAK_H
+#define __MEMLEAK_H
+
+#define MAX_HASH_ENTRY_NUM 1024*1024
```

Comment:
`(1024 * 1024)` ?

Reply:
Fix in commit [2beb289e](https://github.com/iovisor/bcc/pull/3828/commits/2beb289e98d651bddc2127299c5627b5d6149fce).